cypress/integration/tests/controls/businessservices/crud.test.ts

```diff
@@ -19,21 +16,23 @@ describe("A single Business service", () => {
         cy.intercept("GET", "/api/controls/business-service*").as("getBusinessService");
     });
 
-    it("Business service crud operations", function () {
+    it("Business service CRUD operations", function () {
         // Create new Business service
         businessService.create();
         cy.wait("@postBusinessService");
+        exists(businessService.name);
 
         // Edit Business service's name
-        var updatedBusinessServiceName = data.getBusinessServiceName();
+        var updatedBusinessServiceName = data.getCompanyName();
         businessService.edit({ name: updatedBusinessServiceName });
         cy.wait("@getBusinessService");
+        exists(updatedBusinessServiceName);
 
```

Comment:
We should go with `businessService.name` instead of `updatedBusinessServiceName`. 
because it will make sure that the object is updated with a new name after edit.

Reply:
The validation is for name visible on UI. We are anyways updating object inside edit. So no need to change that.